src/docs/resources/faq.md

```diff
@@ -998,4 +999,4 @@ follow Apple's [guidelines][] for App Store submission.
 [web instructions]: /docs/get-started/web
 [`Widget`]: {{site.api}}/flutter/widgets/Widget-class.html
 [widgets]: /docs/development/ui/widgets
-[supported platforms]: /docs/development/tools/sdk/release-notes/supported-platforms
\ No newline at end of file
+[supported platfomrs]: /docs/development/tools/sdk/release-notes/supported-platforms].
```

Comment:
OK....... You introduced a typo here. Also, delete the square bracket at the end.